src/lt/util/dom.cljs

```diff
@@ -37,92 +38,228 @@
+
+  ;; Returns nil, but makes the change.
+  (dom/set-css ($ \"div\") {\"bottom\" \"50px\"})
+  ;;=> nil
+  ;; div is now: <div class=\"tabsets\" style=\"bottom: 50px;\">
+  ```"
+  [elem things]
   (doseq [[k v] things]
     (aset (.-style elem) (name k) (if (keyword? v) (name v) v))))
 
-(defn css [elem things]
+(defn css
+  "If `things` is a map, sets the CSS of `elem` via [[set-css]]. Otherwise,
+  returns what is located at key `things` in `elem`.
+
+  Example:
+  ```
+  ;; Assume there exists a div such as:
+  ;; <div class=\"tabsets\" style=\"bottom: 34px;\">
+
+  (css ($ \"div .tabsets\") :bottom)
+  ;;=> \"34px\"
+
+  ;; Returns nil, but makes the change.
+  (css ($ \"div .tabsets\") {\"bottom\" \"50px\"})
+  ;;=> nil
+
+  (css ($ \"div .tabsets\") \"bottom\")
+  ;;=> \"50px\"
+  ```"
+  [elem things]
   (let [things (if (= js/Object (type things))
-            (js->clj things)
-            things)]
+                 (js->clj things)
+                 things)]
     (if (map? things)
       (set-css elem things)
       (aget (.-style elem) (name things)))))
 
-(defn set-attr [elem things]
+(defn set-attr
+  "Add each key-value pair in `things` to `elem`'s attributes.
+
+  Returns `nil`, if change was successful.
+
+  Example:
+  ```
+  ;; Assume there exists a div such as:
+  ;; <div class=\"tabsets\">
+
+  ;; Returns nil, but makes the change.
+  (dom/set-attr ($ \"div .tabsets\") {:draggable \"true\"})
+  ;;=> nil
+  ;; div is now: <div class=\"tabsets\" draggable=\"true\";>
+  ```"
+  [elem things]
   (doseq [[k v] things]
     (.setAttribute elem (name k) (if (keyword? v) (name v) v))))
 
-(defn attr [elem things]
+(defn attr
+  "If `things` is a map, sets the attributes of `elem` via [[set-attr]]. Otherwise, returns
+  what is located at key `things` in `elem`.
+
+  Example:
+  ```
+  ;; Assume there exists a div such as:
+  ;; <div class=\"tabsets\" draggable=\"true\";>
+
+  (attr ($ \"div .tabsets\") \"draggable\")
+  ;;=> \"true\"
+
+  ;; Returns nil, but makes the change.
+  (attr ($ \"div .tabsets\") {\"draggable\" \"false\"})
+  ;;=> nil
+
+  (attr ($ \"div .tabsets\") \"draggable\")
+  ;;=> \"false\"
+  ```"
+  [elem things]
   (if (map? things)
     (set-attr elem things)
     (.getAttribute elem (name things))))
 
-(defn parent [elem]
+(defn parent
+  "Return the parent node of `elem`.
+
+  Note: using `parent` on the document node will result in a type error.
+
+  Example:
+  ```
+  (parent ($ \"body\"))
+  ;;=> #<[object HTMLHtmlElement]>
+
+  (.-nodeName (parent ($ \"body\")))
+  ;;=> \"HTML\"
+  ```"
+  [elem]
   (.-parentNode elem))
 
-(defn children [elem]
+(defn children
+  "Return the child nodes of `elem`."
+  [elem]
   (.-children elem))
 
-(defn remove [elem]
+(defn remove
+  "Remove `elem` from the DOM tree."
+  [elem]
   (when-let [p (parent elem)]
     (.removeChild p elem)))
 
-(defn empty [elem]
+(defn empty
+  "Sets the inner HTML of `elem` to an empty string."
+  [elem]
   (set! (.-innerHTML elem) ""))
```

Comment:
I suspect you meant the code, in which case, I've updated the [refactor wiki page](https://github.com/LightTable/LightTable/wiki/Possible-Refactoring-Targets) with this info.